data/brands/leisure/fitness_centre.json

```
@@ -13,6 +13,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Econofitness",
+      "locationSet": {"include": ["ca"]},
```

Looking at https://econofitness.ca/en/results?searchmode=searchall&searchtext=&filters= , it seems like their locations are mostly in Quebec with a few in Ontario. If this is correct, I think it would make sense to limit it to those provinces.
```suggestion
      "locationSet": {"include": ["ca-qc.geojson", "ca-on.geojson"]},
```